### ISSUE TITLE
Vimeoplayer is blocked; Error message should cite PB

### ISSUE THREAD

**Issue by textexin** [EFForg/privacybadgerfirefox#609]
A webpage that was using vimeoplayer was blocked.I had to change PB to make vimeo green before it would let the page play.

Also the video is replaced by a message that says sorry because of its privacy settings, the video cannot be played. It would have helped greatly if the message said it came from Privacy Badger.
There are many settings that can interfere with a player, and it wasn't my first thought to look at PB.
![privacy error](https://cloud.githubusercontent.com/assets/14288472/9869234/abd4354a-5b36-11e5-8efe-304037164426.jpg)

**Comment by cooperq** [comment]
Sorry about that! The error message doesn't come from privacy badger so there is no way for us to indicate as such. Can you send me the url of the page that has vimeo blocked so I can take a look and try to fix this issue more generally please?

**Comment by textexin** [comment]
It was a 5 day sale which is over now, so the link contents have changed. That is why I didn’t provide it with the bug report

 

https://5daydeal.com/?ref=9

 

 

Changing PB to force vimeo player to green allowed the video to be played so I think the problem was from PB. Not sure where the message came from if not PB.

 

tex

**Comment by cooperq** [comment]
It was definitely blocked by privacy badger, you are correct about that! But the error message actually comes from vimeo not from privacy badger. 
I am going to close this issue for now since we can't reproduce it, please open a new issue if you find vimeo blocked again.
